app/src/main/java/org/mozilla/focus/settings/ManualAddSearchEngineSettingsFragment.java

```diff
@@ -85,11 +89,13 @@ public void onPause() {
             activeAsyncTask = null;
             menuItemForActiveAsyncTask = null;
         }
+        imm.hideSoftInputFromWindow(getView().getWindowToken(),0);
```

Comment:
nit: Please use `ViewUtils.hideKeyboard`

Reply:
I wanted to use this at first, but using ViewUtils' classes and functions in ManualAddSearchEngineSettingsFragment.java does nothing at all. Do you have any idea why ?

Reply:
Unfortunately, I don't. Have you tried using the debugger to step through the call to see what is (or isn't) being called?